File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/query/QueryRequest.java
package uy.com.searchandfind.query;

import uy.com.searchandfind.Request;

/**
 * Created by strat on 24/10/2017.
 */

public class QueryRequest extends Request {
    private String QueryId;
    private String Category ;
    private String Descritpion;
    private double Latitude ;
    private double Length;

    public String getCategory() {
        return Category;
    }

    public void setCategory(String category) {
        Category = category;
    }

    public String getDescritpion() {
        return Descritpion;
    }

    public void setDescritpion(String descritpion) {
        Descritpion = descritpion;
    }

    public double getLatitude() {
        return Latitude;
    }

    public void setLatitude(double latitude) {
        Latitude = latitude;
    }

    public double getLength() {
        return Length;
    }

    public void setLength(double length) {
        Length = length;
    }

    public String getQueryId() {
        return QueryId;
    }

    public void setQueryId(String queryId) {
        QueryId = queryId;
    }
}

File: SearchAndFindBackend/SearchAndFind.DTO/Query/QueryRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class QueryRequest:Request
    {
        public string Category { get; set; }       
        public string Descritpion { get; set; }
        public double Latitude { get; set; }
        public double Length { get; set; }
        public string QueryId { get; set; }
        public string TenderConfirmId { get; set; }
        
    }
}
